### PR TITLE
feat(dws): add datsource to get resource quotas of dws

### PR DESCRIPTION
**Pull request by wuzhuanhong**
**What this PR does / why we need it**:

Add new datasource to get the list of the resource quotas under the DWS.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add new datasource.
2. add related document and acceptance test.
```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
./scripts/acc-test.sh 
>>> Start to test
=== RUN   TestAccDataSourceQuotas_basic
=== PAUSE TestAccDataSourceQuotas_basic
=== CONT  TestAccDataSourceQuotas_basic
--- PASS: TestAccDataSourceQuotas_basic (19.62s)
PASS
```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: docs/data-sources/dws_quotas.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  The [quotas](#quotas_quotas_struct) structure is documented below.
+
+<a name="quotas_quotas_struct"></a>
+The `quotas` block supports:
+
+* `type` - The type of the quota.
+
+* `quota` - The number of available quotas.
+
+* `used` - The number of quota used.
```
Comment (ruwenqiang123) [inline docs/data-sources/dws_quotas.md]:
```suggestion
* `used` - The number of quotas used.
```

**Comment by wuzhuanhong** [comment]
make testacc TEST='./huaweicloud/services/acceptance/dws' TESTARGS='-run TestAccDataSourceQuotas_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccDataSourceQuotas_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceQuotas_basic
=== PAUSE TestAccDataSourceQuotas_basic
=== CONT  TestAccDataSourceQuotas_basic
--- PASS: TestAccDataSourceQuotas_basic (17.76s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       17.820s

**Comment by wuzhuanhong** [comment]
make testacc TEST='./huaweicloud/services/acceptance/dws' TESTARGS='-run TestAccDataSourceQuotas_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccDataSourceQuotas_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceQuotas_basic
=== PAUSE TestAccDataSourceQuotas_basic
=== CONT  TestAccDataSourceQuotas_basic
--- PASS: TestAccDataSourceQuotas_basic (19.62s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       19.739s

**Comment by wuzhuanhong** [comment]
make testacc TEST='./huaweicloud/services/acceptance/dws' TESTARGS='-run TestAccDataSourceQuotas_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccDataSourceQuotas_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceQuotas_basic
=== PAUSE TestAccDataSourceQuotas_basic
=== CONT  TestAccDataSourceQuotas_basic
--- PASS: TestAccDataSourceQuotas_basic (17.45s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       17.499s

**Comment by wuzhuanhong** [comment]
make testacc TEST='./huaweicloud/services/acceptance/dws' TESTARGS='-run TestAccDataSourceQuotas_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccDataSourceQuotas_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceQuotas_basic
=== PAUSE TestAccDataSourceQuotas_basic
=== CONT  TestAccDataSourceQuotas_basic
--- PASS: TestAccDataSourceQuotas_basic (17.66s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       17.826s

**Comment by wuzhuanhong** [comment]
make testacc TEST='./huaweicloud/services/acceptance/dws' TESTARGS='-run TestAccDataSourceQuotas_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dws -v -run TestAccDataSourceQuotas_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceQuotas_basic
=== PAUSE TestAccDataSourceQuotas_basic
=== CONT  TestAccDataSourceQuotas_basic
--- PASS: TestAccDataSourceQuotas_basic (17.76s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws       17.816s

**Pull request merged**